server/src/main/java/com/linecorp/centraldogma/server/internal/mirror/GitMirror.java

```diff
@@ -240,8 +239,8 @@ protected void mirrorRemoteToLocal(File workDir, CommandExecutor executor,
                     final byte[] content = reader.open(objectId).getBytes();
                     switch (EntryType.guessFromPath(localPath)) {
                         case JSON:
-                            final JsonNode jsonNode = Jackson.readTree(content);
-                            changes.putIfAbsent(localPath, Change.ofJsonUpsert(localPath, jsonNode));
+                            final String jsonText = new String(content, StandardCharsets.UTF_8);
+                            changes.putIfAbsent(localPath, Change.ofJsonUpsert(localPath, jsonText));
```

Comment:
How about also adding the variant method that takes `byte[]` so that we can remove String conversion?
```java
static Change<JsonNode> ofJsonUpsert(String path, byte[] jsonBytes) {...}
```